app/react/Attachments/components/specs/FileList.spec.tsx

```diff
@@ -37,20 +38,37 @@ describe('FileList', () => {
     };
   });
 
-  const render = () => {
+  const render = (props: FileListProps) => {
     component = shallow(<FileList {...props} />);
   };
 
-  it('should render the files', () => {
-    render();
+  it('should render the files correctly', () => {
+    render(props);
     const renderedFiles = component.find(File);
     expect(renderedFiles.length).toBe(2);
     expect(renderedFiles.at(0).props().file).toBe(file);
     expect(renderedFiles.at(1).props().file).toBe(file2);
+    const firstFile = renderedFiles.at(0).props().file;
+    const language = languageLib.get(firstFile.language as string, 'ISO639_1');
+    expect(entity.language).toEqual(language);
+  });
+
+  it('should render the files starting with the one with the system language', () => {
+    const firstFileLanguage = languageLib.get(props.files[0].language as string, 'ISO639_1');
+    if (firstFileLanguage === entity.language) {
+      const temp = props.files[0];
+      props.files[0] = props.files[props.files.length - 1];
+      props.files[props.files.length - 1] = temp;
+    }
```

Comment:
@grafitto i do not think this is needed, you can just redefine props, its shorter and with better readability.
```
 props.files = [file2, file];
```
 i think it is also a good idea to try to avoid conditionals on tests as much as possible.

Reply:
@daneryl done.